Makefile

```diff
@@ -1,9 +1,10 @@
+#VERSION=$(shell cat VERSION)
+VERSION=1.3.0
```

Comment:
which one is supposed to be the correct one to use? (`VERSION` file still exists)

Reply:
Fixed.  The version number defined in the VERSION file is the one used.